resources/measures/upgrade_df_thermostat_control_load_shed/measure.rb

```diff
@@ -84,12 +84,23 @@ def arguments(model)
     num_timesteps_in_hr.setDefaultValue(4)
     args << num_timesteps_in_hr
 
-    choices = ['full baseline', 'bin sample', 'part year bin sample']
+    choices = ['full baseline', 'bin sample', 'part year bin sample', 'fix', 'oat']
```

Comment:
Do you have the measure options described in the measure or parameter descriptions?